packages/mds-stream/kafka/read-stream.ts

```diff
@@ -0,0 +1,18 @@
+import Kafka, { ConsumerStream } from 'node-rdkafka'
+import { StreamReader } from '../stream-interface'
+import { killReadStream, createReadStreamWrapper } from './helpers'
+
+export const KafkaStreamReader: (name: string, readCb: (data: Kafka.ConsumerStreamMessage) => void) => StreamReader = (
+  name,
+  readCb
+) => {
+  let stream: ConsumerStream | undefined
+  return {
+    initialize: async () => {
+      stream = await createReadStreamWrapper({}, { topics: [name] }, readCb)
```

Comment:
This assignment should probably be within a `if (!stream)` conditional.

Reply:
Good call